lib/ansible/modules/group.py

```diff
@@ -244,6 +251,28 @@ def group_info(self):
         return info
 
 
+# ===========================================
+
+class Linux(Group):
+    """
+    This is a Linux Group manipulation class. This is to apply the '-f' parameter to the groupdel command
+
+    This overrides the following methods from the generic class:-
+        - group_del()
+    """
+
+    platform = 'Linux'
+    distribution = None
+
+    def group_del(self):
+        if self.local:
+            command_name = 'lgroupdel'
+        else:
+            command_name = 'groupdel'
+        cmd = [self.module.get_bin_path(command_name, True), '-f' if self.force else None, self.name]
```

Comment:
Although it probably works passing `None` instead of "nothing" for the no force case makes me a bit nervous :)

Reply:
Me too, but setting it to `else ''` causes it to break. I think `None` is the best solution here as it just gets ignored when setting out the cmd.

Reply:
I think that relies on too much internal implementation details, and could be problematic down the line, if some change was made to not silently pop `None` from commands being executed.

Should probably do:

```
cmd = [self.module.get_bin_path(command_name, True)]
if self.force:
    cmd.append('-f')
cmd.append(self.name)
```

Reply:
That works, I'll update to that